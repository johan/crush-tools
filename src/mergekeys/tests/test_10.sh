test_number=10
description="mergefield default value"

for i in `seq 0 ${#test_variants[*]}`; do
  outfile="$test_dir/test_$test_number.${test_variants[$i]}.actual"
  expected=$test_dir/test_$test_number.${test_variants[$i]}.expected
  $bin ${test_variants[$i]} \
       -D 'N/A' -o "$outfile" \
       "$test_dir/test_$test_number.a" \
       "$test_dir/test_$test_number.b"

  if [ $? -ne 0 ] ||
     [ "`diff -q $outfile $expected`" ]; then
    test_status $test_number $i "$description" FAIL
  else
    test_status $test_number $i "$description" PASS
    rm "$outfile"
  fi
done
