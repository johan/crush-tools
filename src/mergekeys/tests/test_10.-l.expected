Key	Col A	Col B 
430987	A1	B2
430987	A2	B2
431000	A3	N/A
431001	A3.1	N/A
431002	A3.2	N/A
431003	A3.3	N/A
431087	A4	B3
439999	A5	N/A
